From b10b7836df09e96e0ba0b3e082933695fa44fc85 Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Wed, 06 Nov 2024 16:48:41 +0200
Subject: [PATCH] [FSW-10975] fs1: align with p1p28 header

Change name of CXL_EP_PCIE PMMU_BASE according to
p1p28 spec header

Change-Id: Ie7c9ebe553b79726320b43e26d8259e56093183e
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index adab2c4..f226681 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -191,26 +191,26 @@
 {
 	u32 val;
 
-	regmap_writel(HBW_DRAIN_TIMEOUT, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+	regmap_writel(HBW_DRAIN_TIMEOUT, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 					 mmPMMU_PCIE_WRAP_HBW_DRAIN_TIMEOUT);
 	regmap_writel(PMMU_PCIE_WRAP_HBW_RSP_ERR_DRAIN_STAMP_EN_M,
-		      mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+		      mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 		      mmPMMU_PCIE_WRAP_HBW_RSP_ERR_DRAIN_STAMP);
-	val = regmap_readl(mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+	val = regmap_readl(mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 			   mmPMMU_PCIE_WRAP_HBW_DRAIN_CFG);
 	val |= PMMU_PCIE_WRAP_HBW_DRAIN_CFG_EN_M;
-	regmap_writel(val, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+	regmap_writel(val, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 			   mmPMMU_PCIE_WRAP_HBW_DRAIN_CFG);
 
-	regmap_writel(LBW_DRAIN_TIMEOUT, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+	regmap_writel(LBW_DRAIN_TIMEOUT, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 					 mmPMMU_PCIE_WRAP_LBW_DRAIN_TIMEOUT);
 	regmap_writel(PMMU_PCIE_WRAP_LBW_RSP_ERR_DRAIN_STAMP_EN_M,
-		      mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+		      mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 		      mmPMMU_PCIE_WRAP_LBW_RSP_ERR_DRAIN_STAMP);
-	val = regmap_readl(mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+	val = regmap_readl(mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 			   mmPMMU_PCIE_WRAP_LBW_DRAIN_CFG);
 	val |= PMMU_PCIE_WRAP_LBW_DRAIN_CFG_EN_M;
-	regmap_writel(val, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_CFG_BASE +
+	regmap_writel(val, mmIO_0_CFG_HALF_0_CXL_EP_PCIE_PMMU_PMMU_PCIE_WRAP_BASE +
 			   mmPMMU_PCIE_WRAP_LBW_DRAIN_CFG);
 }
 
